src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -496,5 +538,27 @@ private void UpdatedBy(Party party, UserInfo userInfo)
                     break;
             }
         }
+
+        public Party PartyApprovals
+        {
```

Comment:
I really like this, but is this function used outside of the test I can see? Also is there more value in the inverse of this function? ie which Parties still need to approve the cohort?

Reply:
yep, can add that in future if it proves to be useful